src/prommis/uky/costing/models/hd_ce.py

```diff
@@ -0,0 +1,1053 @@
+from pyomo.environ import (
+    Constraint,
+    Param,
+    Var,
+    units,
+    PositiveReals,
+    exp,
+    Any,
+)
+from idaes.core import (
+    declare_process_block_class,
+    FlowsheetCostingBlockData,
+    register_idaes_currency_units,
+)
+from idaes.core.util.constants import Constants
+from idaes.core.util.math import smooth_max
+import idaes.logger as idaeslog
+
+_log = idaeslog.getLogger(__name__)
+
+
+def custom_REE_plant_currency_units():
+    """
```

Comment:
I might consider renaming this file to hydrogen_decrepitation to make it more clear for users